### ISSUE TITLE
WebEssentials Version?

### ISSUE THREAD

**Issue by shellyfld** [ptakes/Pta.Build.WebEssentialsBundleTask#1]
Is it correct to assume that this task works with WebEssentials 2013 and does not support the new setup of WebEssentials 2015 with the standalone Bundler & Minifier?  Thanks!

**Comment by ptakes** [comment]
Yes this is correct. I am working on a solution for this. Stay tuned....
